Makefile

```diff
@@ -44,4 +44,10 @@ node_modules: package.json
 clean:
 	@rm -rf node_modules build
 
+lint:
+	./node_modules/.bin/eslint ./lib
+
+lint-fix:
+	./node_modules/.bin/eslint ./lib --fix
```

Comment:
Let's use `$(BIN)` here, add a `node_modules` dependent target, and move it above the `node_modules` target.